test/commons/aria/get-role.js

```diff
@@ -219,12 +219,23 @@ describe('aria.getRole', function() {
 			var node = fixture.querySelector('#target');
 			assert.equal(aria.getRole(node), 'listitem');
 		});
+
+		it('throws an error if the tree is incomplete', function() {
+			fixture.innerHTML =
+				'<ul role="presentation"><li id="target">foo</li></ul>';
```

Comment:
Because I specifically made the tree have just the `<li>` to show that it will fail even if the parent exists in the DOM.

```js
var node = fixture.querySelector('#target');
flatTreeSetup(node);
```